<?xml version="1.0" encoding="UTF-8"?>

<launch>
        <arg name="v" default="0.7"/>
        <arg name="w" default="-0.2"/>
        <arg name="dx_r" default="9"/>
        <arg name="dx_l" default="2.5"/>
        <arg name="dy_u" default="8"/>
        <arg name="dy_d" default="3"/>
    <node pkg="turtlesim" type="turtlesim_node" name="turtlesim_node"/>
    <node pkg="turtlesim_tutorial" type="move_inside_room.py" name="move_turtle" output="screen" >
        <param name="v" value="$(arg v)"/>
        <param name="w" value="$(arg w)"/>
        <param name="dx_r" value="$(arg dx_r)"/>
        <param name="dx_l" value="$(arg dx_l)"/>
        <param name="dy_u" value="$(arg dy_u)"/>
        <param name="dy_d" value="$(arg dy_d)"/>
    </node>
</launch>